.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 /dev/
 Manifest.toml
+build
```

Comment:
Where does this directory come from? AFAICT it should never be generated.

Reply:
This is the generated documentation's build directory, containing the HTML pages.